docs/content/gds/admin/configuration/bff.en.md

```diff
@@ -1,7 +1,172 @@
+| GDS_BFF_AUTH0_ISSUER         | string   |         | Set to the custom domain if enabled in Auth0 (ensuring the trailing slash is set if required by the Auth0 configuration) - this will confirm the issuer from the Auth0 JWT tokens. |
+| GDS_BFF_AUTH0_AUDIENCE       | string   |         | The audience to verify for the Auth0 API configuration (usually the unique name of the API).                                                                                       |
+| GDS_BFF_AUTH0_PROVIDER_CACHE | duration | 5m      | Configures the JWKS caching provider to fetch public keys from Auth0 for JWT token validation.                                                                                     |
+| GDS_BFF_AUTH0_CLIENT_ID      | string   |         | The Client ID for the management API specified by Auth0.                                                                                                                           |
+| GDS_BFF_AUTH0_CLIENT_SECRET  | string   |         | The Client Secret for the management API specified by Auth0.                                                                                                                       |
+| GDS_BFF_AUTH0_TESTING        | bool     | false   | If true a mock authenticator is used for testing purposes.                                                                                                                         |
+
+### Network Configuration
+
+The network configuration enables the BFF to connect to the GDS Database, Directory API, and Members API for both the MainNet and TestNet networks. The required configuration and configuration values is the same for both networks but the environment variables are prefixed with `GDS_BFF_TESTNET_` and `GDS_BFF_MAINNET_` respectively.
+
+The configuration without the prefixes is specified below, at the end of each section, we will provide an exhaustive list of environment variables that are required to fully configure the BFF for connecting to both the TestNet and MainNet GDS services.
+
+#### GDS Database Connection
+
+The GDS Database connection is a store connection that is similar to the BFF Datbase connection described in a following section. Configuring for the TestNet and MainNet database connections is as follows:
+
+| EnvVar                   | Type   | Default | Description                                                                          |
+|--------------------------|--------|---------|--------------------------------------------------------------------------------------|
+| DATABASE_URL             | string |         | Required, the DSN to connect to the database on (see below for details)              |
+| DATABASE_REINDEX_ON_BOOT | bool   | false   | When the server starts, instead of loading indexes from disk, recreate and save them |
```

Comment:
```suggestion
| DATABASE_REINDEX_ON_BOOT | bool   | false   | When the server starts, instead of loading indexes from disk, recreate and save them. |
```